### ISSUE TITLE
svelte.js not found (appeared after 0.2.2)

### ISSUE THREAD

**Issue by jfmario** [livebud/bud#226]
To reproduce: go to a pre-existing project, install the latest release of bud, and run `bud build`.

My error is:

```
../go/pkg/mod/github.com/livebud/bud@v0.2.2/framework/view/ssr/ssr.go:279:12: pattern svelte.js: no matching files found
```

The offending part of ssr.go is:

```go
//go:embed svelte.js
var svelteRuntime string
```

There is no svelte.js in that directory, and I don;t think there was before. It used to be `//go:embed svelte.ts`.

I'd do a PR, but I'm thinking you made this change for a reason and something didn't make it into the repo?

**Comment by jfmario** [comment]
I don't have this error when running `go run main.go -C ../bud-sample-1/ build`. My local bud repo got its svelte.js at some point.

**Comment by matthewmueller** [comment]
Ah shoot. I knew there was a reason for keeping these files in source control. 

Ideally, they stay out of source control, but get embedded during release. I'm not sure how to do that right now though. 

Either way, I'll get this fixed one way or another tonight and cut a new release. Sorry about this regression! 

I've also been meaning to add some basic black-box tests in the Makefile, similar to what [esbuild does](https://github.com/evanw/esbuild/blob/fd13718c6195afb9e63682476a774fa6d4483be0/Makefile#L102) that we can run in CI.

**Comment by pacholoamit** [comment]
Hello! Was facing the same issue but when I ran the script with the `VERSION` above it works!

**Comment by matthewmueller** [comment]
Fixed in v0.2.3! Upgrade with:

```
curl -sf https://raw.githubusercontent.com/livebud/bud/main/install.sh | sh
```

Sorry for the trouble!
